help/C/Help_ch_Transactions.xml

```diff
@@ -1105,116 +1502,544 @@ Author:
             <row>
+              <entry></entry>
               <entry><?dbhtml bgcolor="#FF4040"?><?dbfo bgcolor="#FF4040"?>Brown1</entry>
-              <entry><?dbhtml bgcolor="#FF4040"?><?dbfo bgcolor="#FF4040"?>Matches but not reconciled: &ndash; Intervention required, will not be imported.</entry>
+              <entry><?dbhtml bgcolor="#FF4040"?><?dbfo bgcolor="#FF4040"?>Transaction has poor matches and no action was set: &ndash;
+                Not to be imported. Intervention required.
+              </entry>
             </row>
             <row>
-              <entry><?dbhtml bgcolor="#6495ED"?><?dbfo bgcolor="#6495ED"?></entry>
+              <entry></entry>
               <entry><?dbhtml bgcolor="#6495ED"?><?dbfo bgcolor="#6495ED"?>Cornflower Blue</entry>
               <entry><?dbhtml bgcolor="#6495ED"?><?dbfo bgcolor="#6495ED"?>Row is selected</entry>
             </row>
           </tbody>
         </tgroup>
-      </table>
-      <note><para>The colors illustrated here are the defaults for GnuCash specified in <ulink url="https://github.com/Gnucash/gnucash/blob/maint/gnucash/gnucash-fallback-310.css">gnucash-fallback-310.css</ulink>. You may experience different colors if the gtk-3.0 CSS files have been modified</para>
+      </table>    
+      <note><para>The colors illustrated here are the defaults for &app; specified in
+        <ulink url="https://github.com/Gnucash/gnucash/blob/maint/gnucash/gnucash-fallback-310.css">gnucash-fallback-310.css</ulink>.
+        You may experience different colors if the gtk-3.0 CSS files have been modified</para>
       </note>
-      <para>Where a row has been matched to an existing transaction or is to update an existing transaction or a transfer account 
-         has been matched and supplied, double clicking on the row will bring up a dialog which displays detail of the imported 
-         transaction and the transaction in GnuCash to which it has been matched to allow the user to vet the match and correct it,
-         if they feel the wrong transfer account has been assigned or the matcher has not produced a valid result. You may change the
-         checkbox selection if you disagree with the matcher decisions and you may reassign an assigned transfer account (see below).
+      <para>You may always intervene manually to:
+        <itemizedlist>
+          <listitem> change the import action by selecting another checkbox (or none); and/or</listitem>
+          <listitem>manually assign a different transfer account;</listitem>
+        </itemizedlist>
+        after reviewing the matched transactions and accounts and the automatically selected actions.
       </para>
- 
+      <para>Multi row selection is enabled for the list of downloaded transactions. The following operations can be performed on the list of downloaded transactions:</para>
+      <itemizedlist>
+        <listitem><emphasis role="bold"><phrase>Single<mousebutton>Left Click</mousebutton> on a row</phrase></emphasis>
```

Comment:
How about 
`<keycombo action="click"><mousebutton>Button1</mousebutton></keycombo>`,
`<keycombo action="double-click"><mousebutton>Button1</mousebutton></keycombo>`, ...
See examples in docbooks keycombo page.